Libraries/LibJS/Runtime/Intl/DurationFormat.cpp

```diff
@@ -262,7 +262,7 @@ i8 duration_sign(DurationRecord const& duration)
 // 1.1.6 GetDurationUnitOptions ( unit, options, baseStyle, stylesList, digitalBase, prevStyle, twoDigitHours ), https://tc39.es/proposal-intl-duration-format/#sec-getdurationunitoptions
 ThrowCompletionOr<DurationUnitOptions> get_duration_unit_options(VM& vm, String const& unit, Object const& options, StringView base_style, ReadonlySpan<StringView> styles_list, StringView digital_base, StringView previous_style, bool two_digit_hours)
 {
-    // 1. Let style be ? GetOption(options, unit, string, stylesList, undefined).
+    // 1. Let style be ? GetOption(options, unit, STRING, stylesList, undefined).
```

Comment:
Out of curiosity, is there a particular way you need to copy paste the spec comment for it to keep the changed upper-casing for enums in the spec? Whenever I copy paste, it drops the casing of the enum

Reply:
Nope, it's manual :(

Reply:
Ah, dang! I could have sworn back when this change to the spec was first made, the copy paste did work! But yeah, now it doesn't seem to :(